packages/devtools/devtools-extension/src/playwright/fixtures.ts

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { test as base, chromium, webkit, BrowserContext } from '@playwright/test';
+import path from 'path';
+
+const extensionPath = path.join(__dirname, '../../out/devtools-extension'); // make sure this is correct
+
+// https://playwright.dev/docs/chrome-extensions
+export const test = base.extend<{
+  context: BrowserContext;
+  extensionId: string;
+}>({
+  context: async ({ browserName }, use) => {
+    if (browserName === 'firefox') {
+      return;
+    }
+    const browserTypes = { chromium, webkit };
+    const launchOptions = {
+      devtools: true,
+      headless: false,
+      args: [`--disable-extensions-except=${extensionPath}`, `--load-extension=${extensionPath}`],
```

Comment:
I think we could wire up a way for these args to be passed to the @dxos/test browser launcher